nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/scheduling/TestStandardProcessScheduler.java

```diff
@@ -633,7 +633,9 @@ public void testProcessorTimeOutNoResponseToInterrupt() throws InterruptedExcept
         procNode.performValidation();
         scheduler.startProcessor(procNode, true);
 
```

Comment:
Should there be a max number of sleeps so that this may not loop forever?  It may not happen now, but in the future it may be possible after a change right?

Reply:
yes. the test times out.

Reply:
any you can tell that is where you timed out?  ok.  just a knee jerk reaction to the pattern.